### PR TITLE
fix endpoint cannot retrieve bug.

### LINKED ISSUE #1147: egctl cannot retrieve endpoint config
```bash
kubectl exec -n envoy-gateway-system envoy-gateway-57b7d8bb9c-kkhvs -it -- egctl c proxy e envoy-default-eg-63636f73-78f9b7cfdd-676kd
error: unknown resourceType endpoint
command terminated with exit code 1
```

### PR DESCRIPTION
**Pull request by qicz**
fixes #1147

### CODE REVIEW

File: internal/cmd/egctl/config.go [hunk 1]
```diff
@@ -418,7 +418,7 @@ func extractConfigDump(fw kube.PortForwarder) (*adminv3.ConfigDump, error) {
 }
 
 func configDumpRequest(address string) ([]byte, error) {
-	req, err := http.NewRequest("GET", fmt.Sprintf("http://%s/config_dump", address), nil)
+	req, err := http.NewRequest("GET", fmt.Sprintf("http://%s/config_dump?include_eds", address), nil)
```
Comment (zirain) [inline internal/cmd/egctl/config.go:421]: include_eds is expensive, what about just request when you're retrieving endpoint?

Comment (qicz) [reply internal/cmd/egctl/config.go:421]: reference this
 https://www.envoyproxy.io/docs/envoy/latest/operations/admin.html#get--config_dump?include_eds

Comment (qicz) [reply internal/cmd/egctl/config.go:421]: I will optimize this by ep retrieve

**Comment by qicz** [comment]
@zirain pls review again

**Review by zirain — APPROVED** [review]
Thanks!

**Comment by arkodg** [comment]
@qicz thanks for fixing this !
is there any testdata addition needed as well https://github.com/envoyproxy/gateway/tree/main/internal/cmd/egctl/testdata/config ?

**Comment by qicz** [comment]
> @qicz thanks for fixing this ! is there any testdata addition needed as well https://github.com/envoyproxy/gateway/tree/main/internal/cmd/egctl/testdata/config ?

did it @arkodg pls review

**Review by arkodg — APPROVED** [review]
LGTM thanks @qicz !

**Comment by arkodg** [comment]
I see some failing tests @qicz

**Comment by qicz** [comment]
> I see some failing tests @qicz

some lint case fixed, pls review again @arkodg

**Review by arkodg — APPROVED** [review]

**Review by zirain — APPROVED** [review]
Thanks!

**Pull request merged**